lib/linguist/languages.yml

```diff
@@ -272,6 +272,8 @@ ApacheConf:
 Apex:
   type: programming
   extensions:
+  - ".apxc"
+  - ".apxt"
```

Comment:
I downloaded the 221 files associated with that extension using Harvester and counted 106 repositories by 96 users. This is not enough to mandate addition to Linguist by our standards (i.e., hundreds of repositories). I think we should remove it from this pull request and reconsider it in the future if usage grows.

Reply:
Ok, this is a low number of repositories, but it is the official extension of that language. Users who are using the java extension .cls, is because they want them indexed with liguist, but I'm sure they want to send a pull like this or they don't know how. I really think this would help a lot. Thanks for considering this.

https://trailhead.salesforce.com/en/content/learn/modules/developer_console/developer_console_source_code
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_intro.htm

Reply:
Any update? Thank you